src/components/GetMapTrack/GetMapTrack.tsx

```diff
@@ -62,6 +63,15 @@ const GetMapTrack = () => {
           // setFiles={setFiles}
           clearFiles={clearFiles}
         />
+        {tracks.map((path, id) => {
```

Comment:
Looks like the mapping does create bullet-points with new track data, but doesn't actually populate them with any text. We'll have to fix that before we get this merged!

Reply:
Be sure to check out the preview deploy for what I mean by this: https://deploy-preview-57--asyncracing.netlify.app/